cypress/utils/utils.ts

```diff
@@ -45,6 +45,7 @@ export function selectItemsPerPage(items: number): void {
         .then(($toggleBtn) => {
             if (!$toggleBtn.eq(0).is(":disabled")) {
                 $toggleBtn.eq(0).trigger("click");
+                cy.wait(50000);
```

Comment:
This button click opens the page menu, I don't think any wait is required here. And that too 50 seconds

Reply:
I agree with you. It is not happening always. I executed without it. And tests passed.